core/isolate.rs

```diff
@@ -257,6 +262,27 @@ impl Isolate {
     self.dispatch = Some(Arc::new(f));
   }
 
+  pub fn register_op(&mut self, name: &str, op: Box<CoreOpHandler>) -> OpId {
+    self.op_registry.register_op(name, op)
+  }
+
+  pub fn call_op(
+    &self,
+    op_id: OpId,
+    control: &[u8],
+    zero_copy_buf: Option<PinnedBuf>,
+  ) -> CoreOp {
+    if op_id == 0 {
```

Comment:
Temporary hack to easily fetch op map without adding binding in `libdeno`

Reply:
I don’t understand your comment? Can you elaborate?

Reply:
Ideally we want to register `get_ops_map` as an op using `register_op` method. 
```rust
...
let mut op_registry = OpRegistry::new();
op_registry.register_op("get_ops_map", |_control, _zero_copy| {
  // can't pass borrow checker
  let op_map = op_registry.get_op_map();
  let op_map_json = serde_json::to_string(&op_map).unwrap();
  let buf = op_map_json.as_bytes().to_owned().into_boxed_slice();
  Op::Sync(buf)
})
... 

Isolate {
...
op_registry,
}
```

This method would be placed in `op_registry` and reference that registry and I can't make types work

Reply:
For now I just used this hack and placed and registered a fake method in `OpRegistry` so ids are properly aligned

Reply:
Please add a comment here explaining that op_id 0 has special meaning to retrieve name => op_id map

Reply:
Also add a comment in the triple slash rust doc of `Isolate::set_dispatch()` regarding the special meaning of op_id 0 and to `Isolate::register_op()`

Reply:
I changed API a bit and moved all logic related to `ops` to `core/ops.rs`. So this `op_id === 0` situation is contained in that single file.